apps/crn-server/src/data-providers/reminders.data-provider.ts

```diff
@@ -500,7 +500,7 @@ const getEventMaterialsRemindersFromQuery = (
 
 const getSortDate = (reminder: ReminderDataObject): DateTime => {
   if (reminder.entity === 'Research Output') {
-    return DateTime.fromISO(reminder.data.addedDate);
+    return DateTime.fromISO(reminder.data.addedDate || '');
```

Comment:
I'm unsure how reminders handle draft research outputs, but I think we need some new type to split the ResearchOutput into DraftResearchOutput (which has not added date) and PublishedResearchOutput (which has added date).

Reply:
I think we should discuss this at the RTM @peterstarling, I feel that since we have a lot of places where we only get published ROs and other places where we only get drafts, and there is a way to diff between them (one having an added date and the other not having it) it might warrant some discussion.